sherpa/models/model.py

```diff
@@ -488,6 +489,116 @@ def cache_model(cls, pars, xlo, *args, **kwargs):
     return cache_model
 
 
+# This shares a lot of the code with modelCacher1d, but
+# it's annoyingly hard to factor out the common code, so it's not
+# done here.
+
+def modelCacher1d_exp(func: Callable) -> Callable:
+    """A decorator to cache 1D ArithmeticModel evaluations of a specific form.
+
+    Apply to the `calc` method of a 1D model to allow the model
+    evaluation to be cached. The decision is based on the
+    `_use_caching` attribute of the cache along with the `integrate`
+    setting, the evaluation grid, parameter values, and the keywords
+    sent to the model.
+
+    Unlike the more general `sherpa.models.model.modelCacher1d`, this
+    version is designed for models with a single parameter and a model of
+    the following form:
+
+    math::
+        y(x) = \exp(p * a(x))
+
+    where `p` is the parameter. The cacher will evaluate the model once
+    with `p=1` and then store that. This is useful in particular for
```

Comment:
```suggestion
    with `p=0.1` and then store that. This is useful in particular for
```

I think this is the case. We *could* make the normalization point an argument to the routine (in some manner), but I don't suggest we do that until we decide it's needed.

Reply:
There is also the thought that maybe `0.1` can still lead to 0's in some cases (or is that not really an issue)?

Reply:
If you look at the plots I show with `tbabs` I still get 0's around 0.1 keV when `nh = 0.1` - which might suggest we need to use a smaller value (although I may have used `tbabs` incorrectly as I didn't `set_xsabund('wilm')` or whatever I should have done.

Reply:
See my general comment - this might need more complexity. If we go significantly below 0.1 we don't cover the high-energy end any longer. but that can be done, just at the cost of some complexity.